WcaOnRails/app/webpacker/components/CompetitionForm/MarkdownEditor.js

```diff
@@ -0,0 +1,132 @@
+import React, { useMemo } from 'react';
```

Comment:
Some of these components can very likely be moved up a directory level to increase reusability.